server/ec2spotmanager/views.py

```diff
@@ -274,20 +275,20 @@ def add_children(node):
 
 
 @deny_restricted_users
-def viewConfig(request, configid):
-    config = get_object_or_404(PoolConfiguration, pk=configid)
+def viewEc2Config(request, configid):
+    config = get_object_or_404(Ec2PoolConfiguration, pk=configid)
```

Comment:
The abstraction is in the wrong place here. The URL and handler should be common, the template should be called `view_ec2.html`, and we should select which template to render based on the specific type of the `PoolConfiguration`.